src/modules/0.0.23/aws_lambda/mappers/lambda_function.ts

```diff
@@ -230,6 +336,27 @@ export class LambdaFunctionMapper extends MapperBase<LambdaFunction> {
     delete: async (es: LambdaFunction[], ctx: Context) => {
       for (const e of es) {
         const client = (await ctx.getAwsClient(e.region)) as AWS;
+        const region = e.region;
+
+        // Update function configuration
```

Comment:
Maybe expand this comment that we are doing the update to avoid network interfaces get locked by aws.